dist-repo-builder/src/main/java/io/ballerina/dist/DistRepoBuilder.java

```diff
@@ -63,9 +63,11 @@ public static void main(String args[]) throws IOException {
         // The following list will contain existing docs from ballerina-lang repo
         List<Path> existingDocs = getExistingDocs(jBalToolsPath.resolve("docs"));
         for (Path bala : balas) {
-            generateDocsFromBala(bala, jBalToolsPath, existingDocs);
-            // following function was put in to validate if bir and jar exists for packed balas
-            valid = valid & validateCache(bala, repo);
+            if (!bala.toString().contains("ballerinai")) {
+                generateDocsFromBala(bala, jBalToolsPath, existingDocs);
+                // following function was put in to validate if bir and jar exists for packed balas
+                valid = valid & validateCache(bala, repo);
+            }
```

Comment:
Shall we use a continue statement below, so that we can clearly separate out the omitting logic?

```suggestion
            if (bala.toString().contains("ballerinai")) {
               continue;
            }
            generateDocsFromBala(bala, jBalToolsPath, existingDocs);
            // following function was put in to validate if bir and jar exists for packed balas
            valid = valid & validateCache(bala, repo);
```

Reply:
@NipunaRanasinghe addressed the suggestions.